test/tools/isoltest.cpp

```diff
@@ -90,11 +89,53 @@ string SyntaxTestTool::editor;
 
 void SyntaxTestTool::printContract() const
 {
-	stringstream stream(m_test->source());
-	string line;
-	while (getline(stream, line))
-		cout << "    " << line << endl;
-	cout << endl;
+	if (m_formatted)
+	{
+		std::string const& source = m_test->source();
+		if (source.empty())
+			return;
+
+		std::vector<char const*> sourceFormatting(source.length(), formatting::RESET);
+		for (auto const& error: m_test->errorList())
+			if (error.locationStart >= 0 && error.locationEnd >= 0)
+			{
+				solAssert(static_cast<size_t>(error.locationStart) < source.length(), "");
```

Comment:
I wonder if we should use `solAssert` here or just `assert`?

Reply:
I think it shouldn't be `solAssert` as this method is called outside where Solidity specific exceptions are handled.

Reply:
I think we should promote solAssert to be not specific about solidity anymore.

Reply:
Wouldn't the not-solidity-specific version of ``solAssert`` be libdevcore's ``assertThrow``?
``solAssert`` throws ``dev::solidity::InternalCompilerError`` which is rather solidity-specific itself.

For the moment I changed to plain ``assert`` in ``isoltest.cpp``.